web/src/app/views/about-view/about-view.component.html

```diff
@@ -52,6 +52,43 @@ <h3 class="card-title">Schůzky každý týden</h3>
   </div>
 </section>
 
+<!-- About us -->
+
+<section #bosan class="d-print-none">
+    <div class="container">
+      <div class="row">
+        <div class="col-12 text-center">
+          <h1>Kdo jsme?</h1>

```

Comment:
Tyjo nevím, čím to je, ale máš tam totiž nějaký divný mezery. Ony se pak překládají jako tvrdá mezera (`&nbsp;`) a to pak dělá i ten prostor okolo nadpisu. A taky proto se to možná neformátuje hezky.

Zkus to nějak opravit (nevím jak akorát).

Takhle se mi to zobrazuje v mém VSCode:
![image](https://user-images.githubusercontent.com/1273865/214898389-c63d74dd-9f05-40ea-8851-bae08120fe4b.png)

Reply:
Jo, to se mi tam objevilo při nějakém kopírování. A jestli ten přebytečný prostor je tím, tak to je super, to prostě opravím na normální mezery (já jsem i taky říkal, co to je, ale došel jsem k názoru, že to tak při úpravách asi má vypadat)